products/tarantool.md

```diff
@@ -52,10 +57,10 @@ releases:
     latest: "2.2.3"
 
   - releaseCycle: "1.10"
-    release: 2021-12-22
+    release: 2022-04-25
```

Comment:
The release should be the date when the releaseCycle started, e.g. 2.8.0. You changed to the release date of the latest version, please revert.

Reply:
Fixed.